QUICKSTART-VMWARE.md

```diff
@@ -0,0 +1,258 @@
+# Using Bottlerocket as a Kubernetes worker node with VMware
+
+> The [VMware variant](variants/README.md#vmware-k8s-120-vmware-kubernetes-120-node) is currently in preview phase and we're looking for [feedback](https://github.com/bottlerocket-os/bottlerocket#contact-us).
+> Please try it out and let us know what you think!
+
+This quickstart will walk through joining a Bottlerocket VM guest to an existing Kubernetes cluster running in VMware.
+
+## Prerequisites
+
+You must be able to access vSphere, via webUI or some type of client.
+We will use the CLI tool [`govc`](https://github.com/vmware/govmomi/tree/master/govc) to communicate with vSphere in this quide.
+`govc` can use [environment variables or take arguments](https://github.com/vmware/govmomi/tree/master/govc#usage) to specify needed parameters.
+For the purposes of this guide we will assume that the following environment variables are set to the proper values in your environment:
```

Comment:
We also expect `jq` to be installed to work through the examples, and the AWS CLI to work through the optional steps to enable the control container.